File: packages/cli/test/fixtures/ranking-corpora/csharp/DeleteHandler.cs
namespace RankingCSharp;

/// <summary>Removes a user by id.</summary>
public class DeleteHandler
{
    public void DeleteUser(int id)
    {
        if (id <= 0)
        {
            Logger.LogError("DeleteUser", new ArgumentException("id must be positive"));
            return;
        }

        Logger.LogInfo("deleted user");
    }
}

File: packages/cli/test/fixtures/ranking-corpora/php/src/Handlers/DeleteHandler.php
<?php

namespace RankingPhp\Handlers;

use RankingPhp\Util\Logger;

/** Removes a user by id. */
class DeleteHandler
{
    public function deleteUser(int $id): void
    {
        if ($id <= 0) {
            Logger::logError('deleteUser', new \InvalidArgumentException('id must be positive'));
            return;
        }
        Logger::logInfo('deleted user');
    }
}
